website/content/docs/configuration/client.mdx

```diff
@@ -217,27 +217,35 @@ chroot as doing so would cause infinite recursion.
   files on the client host via the `file` function. By default, templates can
   access files only within the [task working directory].
 
-- `max_stale` `(string: "")` - # This is the maximum interval to allow "stale"
+- `max_stale` `(string: "87600h")` - This is the maximum interval to allow "stale"
   data. By default, only the Consul leader will respond to queries. Requests to
   a follower will forward to the leader. In large clusters with many requests,
   this is not as scalable. This option allows any follower to respond to a query,
   so long as the last-replicated data is within this bound. Higher values result
   in less cluster load, but are more likely to have outdated data.
 
-- `wait` `(Code: nil)` - Defines the minimum and maximum amount of time to wait
-  for the Consul cluster to reach a consistent state before rendering a template.
-  This is useful to enable in systems where network connectivity to Consul is degraded,
-  because it will reduce the number of times a template is rendered. This configuration is
-  also exposed in the _task template stanza_ to allow overrides per task.
+- `wait` `(map: { min = "5s" max = "4m" })` - Defines the minimum and maximum amount
+  of time to wait before attempting to re-render a template. By default, Consul Template
+  will loop continually and re-render. If this value is set, Nomad will configure Consul
```

Comment:
> By default, Consul Template

Aren't 5s/4m the defaults?

Reply:
They are. I thought our convention was to put the defaults in these code snippet sections. Is that not the case?

Reply:
Oh hm, maybe I just misread this sentence. I think when I see:

> By default X. If this value is set, Y.

I assume that the value in question is *not* set by default. The phrasing definitely isn't explicitly saying that, so maybe it's just my poor reading comprehension at work. :sweat_smile: 

The preceding and following sentences clear it all up, but maybe these 2 could be rephrased a bit:

> Consul Template re-renders templates whenever rendered variables from Consul, Nomad, or Vault change. However in order to minimize how often tasks are restarted or reloaded, Nomad will configure <rest of the second sentence here>.

My main point being is that I think the `By default...` in the first sentence makes users think _this parameter changes that behavior when it doesn't._ Re-rendering in a loop is just how Consul Template works and these parameters just tweak some timings.

Reply:
Ah ok. That makes sense. I'll stage a new PR with these doc changes.

Reply:
> Aren't 5s/4m the defaults?

They are not. Default max in consul-template 4 times of **Min**imum value not 4 **Min**utes